<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelFitInspectorViewControls</class>
 <widget class="QWidget" name="ModelFitInspectorViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>865</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,4,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="lblFit">
       <property name="text">
        <string>Modelfit:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbFit"/>
     </item>
     <item row="1" column="1">
      <widget class="QmitkSingleNodeSelectionWidget" name="inputNodeSelector" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblInput">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkPlotWidget" name="widgetPlot" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFullPlot">
       <property name="text">
        <string>Full screen plot</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Fit info</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Fit type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lFitType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fit uid:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lFitUID">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Model type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lModelName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lModelType">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResults">
      <attribute name="title">
       <string>Fit parameter</string>
      </attribute>
      <attribute name="toolTip">
       <string>Parameter used for the fit.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QmitkFitParameterWidget" name="fitParametersWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBookmarks">
      <attribute name="title">
       <string>Inspection positions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QmitkInspectionPositionWidget" name="inspectionPositionWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSettings">
      <attribute name="title">
       <string>View settings</string>
      </attribute>
      <attribute name="toolTip">
       <string>Settings for the visualization of fittings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_x">
         <property name="title">
          <string>X-axis scaling:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QRadioButton" name="radioScaleAuto_x">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>automatic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioScaleFixed_x">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>fixed</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelFixMin_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>min:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbFixMin_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-9999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelFixMax_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>max:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbFixMax_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-9999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnScaleToData_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Sets the min and max to the current range of data +-10%</string>
              </property>
              <property name="text">
               <string>Scale to data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_y">
         <property name="title">
          <string>Y-axis scaling:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioScaleAuto">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>automatic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioScaleFixed">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>fixed</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelFixMin">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>min:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbFixMin">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-9999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelFixMax">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>max:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbFixMax">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-9999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnScaleToData">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Sets the min and max to the current range of data +-10%</string>
              </property>
              <property name="text">
               <string>Scale to data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSettings">
         <property name="title">
          <string>Additional input data visibility:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableInputData">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>20</number>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="toolTip">
              <string extracomment="Name of the data"/>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Color</string>
             </property>
             <property name="toolTip">
              <string extracomment="Visibility of the data in diagram"/>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabData">
      <attribute name="title">
       <string>Plot data export</string>
      </attribute>
      <attribute name="toolTip">
       <string>Display and export option for the current data samples, model signals and additinal fitting informations of the plot.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QmitkFitPlotDataWidget" name="plotDataWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkPlotWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPlotWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkFitParameterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkFitParameterWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkInspectionPositionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkInspectionPositionWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkFitPlotDataWidget</class>
   <extends>QWidget</extends>
   <header>QmitkFitPlotDataWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioScaleFixed</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbFixMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioScaleFixed</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbFixMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioScaleFixed</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnScaleToData</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
